### DISCUSSION TITLE
Feedback from trogntuan

### DISCUSSION THREAD

**Discussion by trogntuan1103** [mdbootstrap/TW-Elements#1814] [Kind words & general feedback]
Hello Tailwind Elements team!

I just started using Tailwind Elements and I wanted to give some feedback.
Overall, I think it's a fantastic product and it's really helped me *speed up my web development process*.

Shout out to [Michal](https://twitter.com/ascensus_mdb) & [Dawid](https://twitter.com/dawidadach) for starting this & great thanks to @Trochonovitz @juujisai @Martussky @smolenski-mikolaj for all the work put in the latest release!

Keep coding!

**Comment by filipkappa** [comment]
Thanks @trogntuan1103, but I believe you forgot to include your suggestions ;)
Keep coding!
